buildtools/cli.go

```diff
@@ -307,6 +316,19 @@ func GetCommands() []cli.Command {
 				return npmCmd(c)
 			},
 		},
+		{
+			Name:            "docker",
+			Flags:           cliutils.GetCommandFlags(cliutils.Docker),
+			Description:     docker.GetDescription(),
```

Comment:
Following the change in https://github.com/jfrog/jfrog-cli/pull/1434
```suggestion
			Usage:           docker.GetDescription(),
```